cmd/execution/main.go

```diff
@@ -145,7 +145,7 @@ func main() {
 			flags.StringVar(&triedir, "triedir", datadir, "directory to store the execution State")
 			flags.StringVar(&executionDataDir, "execution-data-dir", filepath.Join(homedir, ".flow", "execution_data_blobstore"), "directory to use for Execution Data blobstore")
 			flags.Uint32Var(&mTrieCacheSize, "mtrie-cache-size", 500, "cache size for MTrie")
-			flags.UintVar(&checkpointDistance, "checkpoint-distance", 40, "number of WAL segments between checkpoints")
+			flags.UintVar(&checkpointDistance, "checkpoint-distance", 20, "number of WAL segments between checkpoints")
```

Comment:
just maybe we should use some const rather than magic number

Reply:
Great point!  I'll open an issue/PR to replace magic numbers in the flags. E.g.

![image](https://user-images.githubusercontent.com/33205765/160664028-a2dd67ba-be83-4f38-8638-fbb6ad4d23f6.png)